values/src/output.rs

```diff
@@ -1,61 +1,93 @@
 use moment::*;
 use dimension::*;
+use rustling::Value;
 
 #[derive(Clone,PartialEq,Debug)]
 pub enum Output {
     Integer(IntegerOutput),
     Float(FloatOutput),
     Percentage(PercentageOutput),
     Ordinal(OrdinalOutput),
-    Time(TimeOutput),
-    TimeInterval(TimeIntervalOutput),
+    Datetime(DatetimeOutput),
+    DatetimeInterval(DatetimeIntervalOutput),
     AmountOfMoney(AmountOfMoneyOutput),
     Temperature(TemperatureOutput),
     Duration(DurationOutput),
 }
 
-impl Output {
-    pub fn kind(&self) -> OutputKind {
-        match self {
-            &Output::Integer(_) => OutputKind::Number,
-            &Output::Float(_) => OutputKind::Number,
-            &Output::Ordinal(_) => OutputKind::Ordinal,
-            &Output::Time(_) => OutputKind::Time,
-            &Output::TimeInterval(_) => OutputKind::Time,
-            &Output::AmountOfMoney(_) => OutputKind::AmountOfMoney,
-            &Output::Temperature(_) => OutputKind::Temperature,
-            &Output::Duration(_) => OutputKind::Duration,
-            &Output::Percentage(_) => OutputKind::Percentage,
-        }
-    }
-}
+// Seems unused - remove?
+//impl Output {
+//    pub fn kind(&self) -> OutputKind {
+//        match self {
+//            &Output::Integer(_) => OutputKind::Number,
+//            &Output::Float(_) => OutputKind::Number,
+//            &Output::Ordinal(_) => OutputKind::Ordinal,
+//            &Output::Datetime(_) => OutputKind::Datetime,
+//            &Output::DatetimeInterval(_) => OutputKind::Datetime,
+//            &Output::AmountOfMoney(_) => OutputKind::AmountOfMoney,
+//            &Output::Temperature(_) => OutputKind::Temperature,
+//            &Output::Duration(_) => OutputKind::Duration,
+//            &Output::Percentage(_) => OutputKind::Percentage,
+//        }
+//    }
+//}
```

Comment:
Yes. This mapping could be used here: an OutputKind is not needed but grouped in a global dimension and then removed after resolution.